src/components/Request/FormFields/LinkedinQualifications.tsx

```diff
@@ -0,0 +1,76 @@
+import { InfoLabel, Checkbox, Text } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import "components/Request/Request.css";
+import { FormField } from "components/Request/NewRequestForm";
+import { ClipboardCheckmarkRegular } from "@fluentui/react-icons";
+
+const options = [
+  { id: "citizenship", text: "U.S. Citizenship" },
+  { id: "clearance", text: "Obtain & maintain Security clearance" },
+  { id: "drugtest", text: "This is designated Drug Testing Position" },
+  { id: "certification", text: "Certifications/Licensure" },
+  {
+    id: "financial",
+    text: "Financial management Statement - OGE-450 Financial Disclosure Report",
```

Comment:
Capitalize Management

```suggestion
    text: "Financial Management Statement - OGE-450 Financial Disclosure Report",
```